[release-2.9] `build-extension-catalog` fails

<!--------- For bugs and general issues --------->
**Setup**
- Rancher version: 2.9.5
- Rancher UI Extensions: legacy-v2
- Browser type & version:

**Describe the bug**
<!--A clear and concise description of what the bug is.-->
Empty `tagged_release` causes trailing error.
```
Run yarn parse-tag-name  12645625456 "catalog"
yarn run v1.22.22
warning package.json: No license field
$ ./node_modules/@rancher/shell/scripts/extension/parse-tag-name 12645625456 catalog
Parse tag name - evaluating release tag 12645625456
release tag doesn't match any chart tag: 12645625456. Check your pkg/<!-YOUR-EXT-> folders and corresponding versions to complete the match
invalid run-id "catalog"
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
Error: Process completed with exit code 1.
```

**To Reproduce**
<!--Steps to reproduce the behavior-->

**Result**

**Expected Result**
<!--A clear and concise description of what you expected to happen.-->

**Screenshots**
<!-- If applicable, add screenshots to help explain your problem and/or errors in the browser's dev console -->

**Additional context**
<!--Add any other context about the problem here. -->

--- a/creators/extension/app/files/.github/workflows/build-extension-catalog.yml
+++ b/creators/extension/app/files/.github/workflows/build-extension-catalog.yml
@@ -20,5 +20,6 @@ jobs:
     with:
       registry_target: ghcr.io
       registry_user: ${{ github.actor }}
+      tagged_release: ${{ github.ref_name }}
     secrets: 
       registry_token: ${{ secrets.GITHUB_TOKEN }}

--- a/creators/extension/package.json
+++ b/creators/extension/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@rancher/create-extension",
   "description": "Rancher UI Extension generator",
-  "version": "3.0.6",
+  "version": "3.0.7",
   "license": "Apache-2.0",
   "author": "SUSE",
   "packageManager": "yarn@4.5.0",

--- a/docusaurus/docs/extensions/known-issues.md
+++ b/docusaurus/docs/extensions/known-issues.md
@@ -19,3 +19,23 @@ To resolve this add the following `resolution` to the root application's `packag
   ...
 }
 ```
+
+- Running `yarn install` might throw the following errors:
+```
+error @aws-sdk/types@3.723.0: The engine "node" is incompatible with this module. Expected version ">=18.0.0". Got "16.20.2"
+error @aws-sdk/util-locate-window@3.723.0: The engine "node" is incompatible with this module. Expected version ">=18.0.0". Got "16.20.2"
+```
+
+To resolve this add the following `resolutions` to the root application's `package.json`:
+```
+{
+  "name": "app-name",
+  "version": "0.1.0",
+  ...
+  resolutions": {
+    "@aws-sdk/types": "3.714.0",
+    "@aws-sdk/util-locate-window": "3.693.0"
+  },
+  ...
+}
+```

